### PR TITLE
Added shortcut for opening examples 

### LINKED ISSUE #626: Add shortcut for opening examples
@domoritz please let me know if I should go forward with this :)

### PR DESCRIPTION
**Pull request by nik72619c**
Fixes #626 , @domoritz Please let me know if any changes are needed on this :)

### CODE REVIEW

File: src/utils/keyboardShortcuts.ts [hunk 1]
```diff
@@ -23,6 +23,11 @@ const shortcuts = [
     mac: '⌘ + Shift + o',
     text: `Navigate to a symbol in the specification`,
     windows: 'Ctrl + Shift + o'
+  },
+  {
+    mac: '⌘ + 3',
```
Comment (domoritz) [inline src/utils/keyboardShortcuts.ts]: Why this combination?

File: src/components/header/renderer.tsx [hunk 2]
```diff
@@ -508,11 +523,8 @@ class Header extends React.PureComponent<Props, State> {
         </section>
 
         <section className="right-section">
-          <PortalWithState closeOnEsc>
+          <PortalWithState closeOnEsc ref="helpModal">
```
Comment (domoritz) [inline src/components/header/renderer.tsx]: String refs are deprecated in react. Can you find a different way? Do you see the deprecation notice in vscode?

File: src/components/header/renderer.tsx [hunk 3]
```diff
@@ -125,16 +128,27 @@ class Header extends React.PureComponent<Props, State> {
...
-  public handleHelpModalToggle(Toggleevent, openPortal, closePortal, isOpen) {
+  public handleAppShortcuts() {
     window.addEventListener('keydown', event => {
       if (
         (event.keyCode === KEYCODES.SINGLE_QUOTE && event.metaKey && !event.shiftKey) || // Handle key press in Mac
         (event.keyCode === KEYCODES.SLASH && event.ctrlKey && event.shiftKey) // Handle Key press in PC
       ) {
-        if (!isOpen) {
-          openPortal();
+        const modalInstance: any = this.refs.helpModal;
```
Comment (domoritz) [inline src/components/header/renderer.tsx]: Let's avoid any.

File: src/components/header/renderer.tsx [hunk 4]
```diff
@@ -125,16 +128,27 @@ class Header extends React.PureComponent<Props, State> {
...
-          closePortal();
+          modalInstance.closePortal();
+        }
+      }
+      // shortcut for opening examples modal
+      if (
+        (event.keyCode === KEYCODES.THREE && event.metaKey && !event.shiftKey) || // Handle key press in Mac
+        (event.keyCode === KEYCODES.THREE && event.ctrlKey && event.shiftKey) // Handle Key press in PC
+      ) {
+        const modalInstance: any = this.refs.examplesModal;
```
Comment (domoritz) [inline src/components/header/renderer.tsx]: Also avoid any here.

File: src/utils/keyboardShortcuts.ts (hunk 1 above)
Comment (nik72619c) [reply src/utils/keyboardShortcuts.ts]: @domoritz could you suggest any other keyboard shortcut for that? I don't have any specific reason for adding `Cmd+3` as one.

Comment (domoritz) [reply src/utils/keyboardShortcuts.ts]: Something with e in it?

File: src/components/header/renderer.tsx (hunk 4 above)
Comment (nik72619c) [reply src/components/header/renderer.tsx]: cool, I will use callback refs instead. Does it sound good ?

Comment (domoritz) [reply src/components/header/renderer.tsx]: Sure. Do you see the deprecation warning?

**Comment by domoritz** [comment]
@nik72619c Can you finish this?

**Comment by nik72619c** [comment]
@domoritz yup, finishing this soon !

**Comment by domoritz** [comment]
I'm closing this pull request. Please make a new one.

**Pull request closed without merging**